src/module/addRemove.js

```diff
@@ -0,0 +1,136 @@
+/* eslint-disable no-loop-func */
+let todoList = [];
+let isEditing = false;
+let todoEdit = null;
+
+const saveData = () => {
+  localStorage.setItem('todoslist', JSON.stringify(todoList));
+};
+
+const getStorageData = () => {
+  const localFormData = JSON.parse(localStorage.getItem('todoslist'));
+  if (localFormData == null) {
+    todoList = [];
+  } else {
+    todoList = localFormData;
+  }
+};
+
+const editTodoList = (todo) => {
+  isEditing = true;
+  todoEdit = todo;
+  const desc = document.querySelector('.taskList');
+  desc.value = todo.description;
+  desc.focus();
+};
+
+const removeTodoList = (indexID) => {
+  todoList.splice((indexID - 1), 1);
+  todoList = todoList.map(
+    (todo, index) => (
+      { completed: todo.completed, description: todo.description, index: index + 1 }
+    ),
+  );
```

Comment:
- [ ] Kindly use those lines to update indexes in the `add` action because it causes the first delete to be wrong because you depend on index of item for delete you need only to update index after each addition process,
![image](https://user-images.githubusercontent.com/24703171/161175725-c6252ca2-0485-4144-86ab-1f0ad7922671.png)